packages/client/src/modals/TransactionSuccessModal.js

```diff
@@ -0,0 +1,134 @@
+
+    const actionType = actionData.type;
+    
+    const NFTName = collectionId?.split(".")[2];
+    const NFTAddress = collectionId?.split(".")[1];
+
+    useEffect(() => {
+        if (actionType === ACTION_TYPES.TRANSFER_NFT) {
+            const getImageURL = async () => {
+                const result = await getNFTReference(NFTName, formatAddress(NFTAddress), recipient, nftId);
+                setImage(result);
+            };
+            getImageURL().catch(console.error);
+        }
+    }, [actionType, NFTName, NFTAddress, nftId, recipient, getNFTReference])
+
+    function getTokenMeta(vaultId) {
+        if (vaultId) {
+            const tokenName = vaultId.split(".")[2];
+            return COIN_TYPE_TO_META[CONTRACT_NAME_TO_COIN_TYPE[tokenName]];
+        }
+    }
+
+    function getSafeContactName(address) {
+        const contact = contacts.find(contact => contact.address === address);
+        return contact?.name;
+    }
+
+    return (
+        <div className="p-5 has-text-black has-text-left">
+            <div className="p-5 success-modal-background">
+                <button className="button flex-1 is-info mb-1">
+                    Sent <ArrowUp className="ml-2 has-text-white" />
+                </button>
+                <div className="pl-4">
+                    {actionType === ACTION_TYPES.TRANSFER_NFT &&
+                        <>
+                            <img className="columns is-vcentered is-multiline is-mobile mr-2 mt-2 success-modal-image" src={imageURI} alt={imageName} />
+                            <span className="columns is-vcentered is-multiline is-mobile mr-2 is-size-2 is-family-monospace">
+                                #{nftId}
+                            </span>
+                            <span className="columns is-vcentered is-multiline is-mobile is-size-6 has-text-weight-bold">
+                                {NFTName}
+                            </span>
+                        </>
+                    }
+                    {actionType === ACTION_TYPES.TRANSFER_TOKEN &&
+                        <>
+                            <span className="columns is-vcentered is-multiline is-mobile mr-2 mt-2 is-size-2 is-family-monospace">
+                                {Number(tokenAmount)}
+                            </span>
+                            <span className="columns is-vcentered is-multiline is-mobile is-size-6 has-text-weight-bold">
+                                {icon} &nbsp; {displayName}
+                            </span>
+                        </>
+                    }
+                </div>
+            </div>
+            <div className="mt-4 border-light-top">
+                <div className="column is-vcentered is-multiline is-mobile border-light-bottom is-flex is-full">
+                    <span className="flex-1 has-text-grey">Sent From</span>
+                    &nbsp;
+                    <div className="flex-1" style={{ position: "relative" }}>
```

Comment:
What is the position relative here for?

Reply:
There were some issues with displaying this in the same line, but now it's resolved so I removed it